src/test/java/integration/DiscountControllerIT.java

```diff
@@ -99,26 +98,26 @@ public void createDiscount() throws Exception {
                         post("/discounts").
                 then().
                         statusCode(SC_OK).
+                        body("fromDate", equalTo(inputJson.get("fromDate"))).
+                        body("toDate", equalTo(inputJson.get("toDate"))).
                 extract().
                         path("uuid");
-        long timeFinished = currentTimeMillis();
+        LocalDateTime timeFinished = now();
         UUID newDiscountUuid = UUID.fromString(newUuid);
 
         // check created data
         JSONParser parser = new JSONParser();
         JSONObject expectedJson = (JSONObject) parser.parse(inputJson.toJSONString());
 
         Discount createdDiscount = discountService.getEntityByUuid(newDiscountUuid);
-        long timeCreated = createdDiscount.getCreated().getTime();
-        long timeModified = createdDiscount.getLastModified().getTime();
+        LocalDateTime timeCreated = createdDiscount.getCreated();
+        LocalDateTime timeModified = createdDiscount.getLastModified();
 
-        assertTrue(WRONG_CREATED_MESSAGE, timeFinished >= timeCreated && timeCreated >= timeStarted);
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_CREATED_MESSAGE, timeFinished.isAfter(timeCreated) && timeCreated.isAfter(timeStarted));
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).